tests/optimize/test_objective.py

```diff
@@ -74,5 +78,152 @@ def tearDown(self):
         self.directory.cleanup()
         del self.directory
 
+class test_RelativeEntropy(unittest.TestCase):
+    """Unit tests for relentless.optimize.RelativeEntropy"""
+
+    def setUp(self):
+        self.directory = tempfile.TemporaryDirectory()
+
+        lj = relentless.potential.LennardJones(types=('1',))
+        self.epsilon = relentless.variable.DesignVariable(value=1.0)
+        self.sigma = relentless.variable.DesignVariable(value=0.9)
+        lj.coeff['1','1'].update({'epsilon':self.epsilon, 'sigma':self.sigma, 'rmax':2.7})
+        self.potentials = relentless.simulate.Potentials(pair_potentials=lj)
+        self.potentials.pair.rmax = 3.6
+        self.potentials.pair.num = 1000
+        self.potentials.pair.fmax = 100.
+
+        v_obj = relentless.volume.Cube(L=10.)
+        self.target = relentless.ensemble.Ensemble(T=1.5, V=v_obj, N={'1':50})
+        rs = np.arange(0.05,5.0,0.1)
+        gs = np.exp(-lj.energy(('1','1'),rs))
+        self.target.rdf['1','1'] = relentless.ensemble.RDF(r=rs, g=gs)
+
+        self.thermo = relentless.simulate.dilute.AddEnsembleAnalyzer()
+        self.simulation = relentless.simulate.dilute.Dilute(operations=[self.thermo])
+
+    def relent_grad(self, var, ext=False):
+        rs = np.arange(0,3.6,3.6/1000)[1:]
```

Comment:
If you use linspace, you can be sure to hit the end point:
```suggestion
        rs = np.linspace(0,3.6,1001)[1:]
```